stats/bin/spark-stats.js

```diff
@@ -20,6 +20,6 @@ const logger = {
 const handler = createHandler({ pgPools, logger })
 const server = http.createServer(handler)
 console.log('Starting the http server on host %j port %s', HOST, PORT)
-server.listen(PORT, HOST)
+server.listen(Number(PORT), HOST)
```

Comment:
what is this cast to number for?

Reply:
The way how TypeScript describes the `listen` method, it accepts only numbers as the port. When we use `PORT`, which can be a string, TypeScript complains.

```js
server.listen('8080', 'localhost')
```